SignalServiceKit/src/Messages/Interactions/TSErrorMessage.m

```diff
@@ -91,6 +91,11 @@ - (instancetype)initWithEnvelope:(OWSSignalServiceProtosEnvelope *)envelope
     return [self initWithTimestamp:envelope.timestamp inThread:contactThread failedMessageType:errorMessageType];
 }
 
+- (OWSInteractionType)interactionType
+{
+    return OWSInteractionType_Error;
```

Comment:
All interactions now expose an `interactionType` which will let us do nice switches & conditions instead of `isKindOfClass:` and improve perf.